ts/nni_manager/core/nnimanager.ts

```diff
@@ -457,6 +457,9 @@ class NNIManager implements Manager {
         } else if (platform === 'local') {
             const module_ = await import('../training_service/local/localTrainingService');
             return new module_.LocalTrainingService(config);
+        } else if (platform === 'kubeflow') {
+            const module_ = await import('../training_service/kubernetes/kubeflow/kubeflowTrainingService');
+            return new module_.KubeflowTrainingService();
```

Comment:
this means kubeflow does not reuse mode by default?

Reply:
This can't be removed, since kubeflow non-reuse mode only support v1 config for now. It can set to reuse mode as default in v2 config nnictl side.